docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+    # sort: 1
+aliases:
+  - /data-ingestion/proxmox.html
+  - /data-ingestion/Proxmox.html
+---
+
+# Proxmox Data Ingestion
+Since Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB We can use the InfluxDB write support built into VictoriaMetrics
+Currently PVE and PBS only support using an Authorization Token for authentication and does not support basic auth or a username and password.
+
+## Proxmox Virtual Environment (PVE)
+If want help Sending your data to Managed VictoriaMetrics check out [our blog](https://victoriametrics.com/blog/proxmox-monitoring-with-dbaas/).
```

Comment:
```suggestion
If you want to send data to Managed VictoriaMetrics check out blog post [Monitoring Proxmox VE via Managed VictoriaMetrics](https://victoriametrics.com/blog/proxmox-monitoring-with-dbaas/).
```

Reply:
I reworded it a different way please review